packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -118,11 +122,77 @@ const makeInvalidTypeError = (
     );
 };
 
-type RenderProps = PerseusInteractiveGraphWidgetOptions; // There's no transform function in exports
+type RenderProps = {
+    // Where the little black axis lines & labels (ticks) should render.
+    // Also known as the tick step. default [1, 1]
+    // NOTE(kevinb): perseus_data.go defines this as Array<number>
+    step: [number, number];
+    // Where the grid lines on the graph will render. default [1, 1]
+    // NOTE(kevinb): perseus_data.go defines this as Array<number>
+    gridStep?: [x: number, y: number];
+    // Where the graph points will lock to when they are dragged. default [0.5, 0.5]
+    // NOTE(kevinb): perseus_data.go defines this as Array<number>
+    snapStep?: [x: number, y: number];
+    // An optional image to use in the background
+    backgroundImage?: PerseusImageBackground;
+    /**
+     * The type of markings to display on the graph.
+     * - graph: shows the axes and the grid lines
+     * - grid: shows only the grid lines
+     * - none: shows no markings
+     */
+    markings: "graph" | "grid" | "none";
```

Comment:
This is slightly OCD, but would you mind making all of these JSDoc-style (`/** */`) comments? Some are, but the ones that aren't won't surface their information at call sites in VSCode and Goland. It's a tiny thing but is handy when accessing the prop in code to see its comment. 

<img width="496" alt="image" src="https://github.com/user-attachments/assets/3de48636-2b04-407d-b3fb-24e0e5e7c99c">